zebedee-cms/src/main/java/com/github/onsdigital/zebedee/session/store/JWTStore.java

```diff
@@ -18,8 +33,21 @@
  */
 public class JWTStore implements Sessions {
 
+    private JWTHandler jwtHandler = new JWTHandlerImpl();
+
     private Map<String, String> rsaKeyMap;
 
+    private static ThreadLocal<UserDataPayload> store = new ThreadLocal<>();
+
+    private final static int JWT_CHUNK_SIZE = 3;
```

Comment:
That value won't change - JWTs are constructed as a three part, `.` delimited string and as this is a JWT verification library, I thought it would be acceptable to provide the array size as a constant.